src/homeScreening.js

```diff
@@ -0,0 +1,50 @@
+import { APIurl, settingsGet } from '../app.js';
+
+// Function to get the screenings for the home page
+async function homeScreening(response, page) {
+  // Fetch the screenings from the API
+  fetch(`${APIurl}${page}`, settingsGet)
+    .then((response) => response.json())
+    .then((json) => {
```

Comment:
You're using old-school promises here, which is inconsistent with how you've chosen to use async/await in other places. It's always prefferrable to be consistent when possible (although there are some situations where you have to use promises).